### ISSUE TITLE
Are types serializable?

### ISSUE THREAD

**Issue by slaymaker1907** [2graphic/sinap-types#3]
[#103](https://github.com/2graphic/sinap-core/issues/103#issuecomment-301348269) would likely need to serialize types without values to communicate with the loader.

**Comment by Sheyne** [comment]
Yeah. But I think only in sinap-core because it's somewhat hacky.
